sdk/core/core-https/src/util/inspect.browser.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export const custom = {};
```

Comment:
Are symbols one of those things we just can't pollyfill in IE11?

Reply:
No, you can at least make a fake polyfill that doesn't quite work the same. We document using one for storage-blob. Do you want this to be a symbol instead of an anonymous object?

Reply:
I don't feel strongly either way, was just curious.